docs/reference/sql/functions/date-time.asciidoc

```diff
@@ -420,6 +420,49 @@ include-tagged::{sql-specs}/docs/docs.csv-spec[dateDiffDateTimeMinutes]
 include-tagged::{sql-specs}/docs/docs.csv-spec[dateDiffDateMinutes]
 --------------------------------------------------
 
+[[sql-functions-datetime-dateformat]]
```

Comment:
For consistency, I'd move this section right after TO_CHAR, so that we have all the formatting functions together (SqlServer, PostgreSQL, MySQL)

Reply:
Well, the functions should be in alphabetical order actually, so this should be fine.